kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -24,9 +24,10 @@
 
         <div class="user-field">
           <label for="user-role"><span class="visuallyhidden">User Role</span></label>
-          <select v-model="role_new" id="user-role">
-            <option :selected="role_new == learner" v-if="role_new" value="learner"> Learner </option>
-            <option :selected="role_new == admin" value="admin"> Admin </option>
+          <select v-model="kind_new" id="user-role">
+            <option :value="LEARNER"> Learner </option>
+            <option :value="ADMIN"> Admin </option>
+            <option :value="COACH"> Coach </option>
```

Comment:
`:selected` is indeed supported. `selected` brings the console errors I was mentioning, and in that console error, they state that defining defaults in `data` is the preferred method:

```
ERROR in ./~/vue-loader/lib/template-compiler.js!./~/svg-inline-loader!./~/vue-loader/lib/selector.js?type=template&index=0!./kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue
    template syntax error <select v-model="kind_new">:
    inline selected attributes on <option> will be ignored when using v-model. Declare initial values in the component's data option instead.
```